packages/api/endpoints/execution-status.js

```diff
@@ -82,9 +86,10 @@ async function get(req, res) {
 
   // get the execution information from database
   let response;
-  const e = new models.Execution();
+  const knex = await getKnexClient({ env: process.env });
+  const executionPgModel = new ExecutionPgModel();
   try {
-    response = await e.get({ arn });
+    response = await executionPgModel.get(knex, { arn });
   } catch (error) {
     if (error instanceof RecordDoesNotExist) {
       return res.boom.notFound('Execution not found in API or database');
```

Comment:
Would it be clearer to return the error that `get` throws? 
`Record in ${this.tableName} with identifiers ${JSON.stringify(params)} does not exist.`

Reply:
I'm assuming this means the text in the res.boom.notFound call, not returning the error itself: e7ff2de35

Reply:
Ah yeah, I did mean the message. Could you get it from `error.message` though rather than adding the same text? The idea was just that we could avoid replacing one perfectly decent error message with another. 

```
if (error instanceof RecordDoesNotExist) {      
  return res.boom.notFound(error.message);
}
```